bindings/wasm/examples/src/0_basic/0_create_did.ts

```diff
@@ -53,7 +53,7 @@ export async function createIdentity(): Promise<{
 
     // Insert a new Ed25519 verification method in the DID document.
     let keypair = new KeyPair(KeyType.Ed25519);
-    let method = new IotaVerificationMethod(document.id(), keypair.type(), keypair.public(), "#key-1");
+    let method = new VerificationMethod(document.id().toCoreDid(), keypair.type(), keypair.public(), "#key-1");
```

Comment:
can't we add a method directly on `IotaDocument` to get the `CoreDID`? something like `id_core()` to avoid chaining here?

Reply:
That is a great idea! (also better for performance).